keyform/templates/keyform/searchtable.html

```diff
@@ -62,7 +62,7 @@
                     </div>
                     <div class="large-6 columns">
                        {% trans 'Status' %} <br/>
-                       <select name='status'>
+                       <select name='status__name'>
                            <option value=""></option>
                            {% for choice in status_types %}
                                <option {% if search_data.status == choice.0 %} selected {% endif %} value="{{ choice.0 }}">{{ choice.1 }}</option>
```

Comment:
You'll need to update this when you change Status' name field to be its actual name.